src/helics/application_api/ValueFederate.cpp

```diff
@@ -104,7 +104,7 @@ void ValueFederate::disconnect()
 ValueFederate& ValueFederate::operator=(ValueFederate&& fed) noexcept
 {
     vfManager = std::move(fed.vfManager);
-    if (getID() != fed.getID()) {  // the id won't be moved, as it is copied so use it as a test if
+    if (getID() != fed.getID()) {  // the inp won't be moved, as it is copied so use it as a test if
```

Comment:
inp == input? `id` seems to make more sense in this comment

```suggestion
    if (getID() != fed.getID()) {  // the id won't be moved, as it is copied so use it as a test if
```

Reply:
change all error I didn't catch